CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+2.0.0-beta6 (in progress)
+=========================
+
+New Features
+------------
+
+- twilio-video.js will now support the Unified Plan SDP format where applicable.
+  Chrome, starting from version 72, and Safari, starting from version 12.1 will
+  enable Unified Plan as the default SDP format. We highly recommend that you
+  upgrade your twilio-video.js dependency to this version so that your application
+  continues to work on the above mentioned browser versions. For more details, please
+  refer to the Unified Plan Guide. (JSDK-2265)
```

Comment:
Having read the rest of it, I'd just keep the first sentence, and move and include the paragraphs of the "Unified Plan Guide" right here, and get rid of that separate section.